packages/cli/src/telemetry/index.ts

```diff
@@ -59,6 +71,9 @@ export default {
 };
 
 async function checkOptIn(interactive: boolean): Promise<GlobalTelemetryOptions | undefined> {
+  // disable via env var for local development
+  if (!!process.env.OPENZEPPELIN_DISABLE_TELEMETRY) return undefined;
```

Comment:
This is causing telemetry to be disabled if the env variable is defined to a value other than the emptry string. Is that the behavior that you wanted?

I'm not a fan of this kind of string to boolean conversion, but there seems to be several of these throughout the codebase so I think I'm okay with keeping it.

Reply:
Still, being a variable used for development, I wouldn't worry too much